api/capi/include/nnstreamer.h

```diff
@@ -38,6 +38,18 @@ extern "C" {
  * @{
  */
 
+/**
+ * @brief The virtual name to set the video source of camcorder in Tizen.
+ * @since_tizen 5.5
+ */
+#define ML_TIZEN_CAM_VIDEO_SRC "TizenCamVideoSrc"
```

Comment:
Are you going to make this case-sensitive??

Reply:
No, it'sTBD :)

Reply:
looks like this macro is not used?

Reply:
1. Changed camsrc name (tizencamvideosrc / tizencamaudiosrc)
2. It is used in capi-tizen to replace src element in pipeline.